app/presenters/finder_content_item_presenter.rb

```diff
@@ -27,14 +27,6 @@ def content_id
     file.fetch("content_id")
   end
 
-  def self.facets_without_specialist_publisher_properties(facets)
```

Comment:
It would be good to do the moving of this into FinderFacetPresenter in a separate commit to then adding the sub_facet_hash logic.

Reply:
Updated now. The refactoring is in a separate commit